backends/cuda-gen/ceed-cuda-gen-operator-build.cpp

```diff
@@ -659,6 +674,38 @@ inline __device__ void gradTranspose3d(BackendData& data, const CeedScalar *__re
   }
 }
 
+template <int NCOMP, int P1d, int Q1d>
+inline __device__ void gradColo3d(BackendData& data, const CeedScalar *__restrict__ r_U,
```

Comment:
Not to be pedantic, but we changed all the `colocated` that were misspelled, to `collocated`, a while ago. This would re-introduce it to the library

Reply:
Thank you, I'll correct the typo.